nemo_skills/inference/merge_chunks.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Remove this one?

Reply:
fixed in [27039e0](https://github.com/NVIDIA/NeMo-Skills/pull/350/commits/27039e00b40e95747efebaea4d99914fbaf7b1ab)